resources/materials/naxe_ABS.xml.fdm_material

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>Naxe</brand>

```

Comment:
Is this really a fundamentally different material? It seems like it's better to just add machine settings to generic 1.75 ABS instead